riscv-asm.md

```diff
@@ -793,7 +793,7 @@ the current position:
 
 [^3]: `ra` is implicitly used to save the return address.
 [^4]: similar to `call <symbol>`, but `<rd>` is used to save the return address instead.
-[^5]: `t1` is implicitly used as a scratch register.
+[^5]: `t2` is implicitly used as a scratch register.
```

Comment:
Don't break backward compatibility, try to add rather than just change:
"`t1` is implicitly used as a scratch register, but if the Zicfiss extension is enabled, it will use t2 instead."